frontend/src/pages/privacy

```diff
@@ -0,0 +1 @@
+../../../node_modules/legal-docs/firefox_testpilot_PrivacyNotice
```

Comment:
Yes -- this was introduced by https://github.com/mozilla/testpilot/pull/2357 and I just moved the symlinks into a different directory. There was discussion about windows at the time, and I did some research which indicated git might not have as many problems with symlinks any more. Since testing these legal docs on windows isn't really that important, it didn't seem like a high priority. I'm still in favor of removing the symlinks if you have a better suggestion -- this just seemed easy. It should probably be done in another bug though.